docstring_parser/tests/test_rest.py

```diff
@@ -338,6 +348,21 @@ def test_returns() -> None:
     assert len(docstring.many_returns) == 1
     assert docstring.many_returns[0] == docstring.returns
 
+    docstring = parse(
+        """
+        Short description
+        :returns: description
+        :rtype: int
+        """
+    )
+    assert docstring.returns is not None
+    assert docstring.returns.type_name == "int"
+    assert docstring.returns.description == "description"
+    assert not docstring.returns.is_generator
+    assert docstring.many_returns is not None
+    assert len(docstring.many_returns) == 1
+    assert docstring.many_returns[0] == docstring.returns
```

Comment:
Can be shortened to `assert docstring.many_returns == [docstring.returns]`

Reply:
Fixed